### PR TITLE
Update global bar text with photo ID guidance

### PR DESCRIPTION
**Pull request by AshGDS**
## What / Why
- Update global bar messaging with photo ID guidance
- https://trello.com/c/4HovWiZO/172-update-wording-of-global-banner-to-go-live-19-june

### CODE REVIEW

**Review by andysellick — APPROVED** [review]

File: app/views/components/_global_bar.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <%
   show_global_bar ||= true # Toggles the appearance of the global bar
 
-  title = "Register to vote"
-  title_href = "/register-to-vote"
-  link_text = "Register by 18 June to vote in the General Election on 4 July."
+  title = "Bring photo ID to vote"
+  title_href = "/how-to-vote/photo-id-youll-need"
+  link_text = "Check what photo ID you'll need to vote in person in the General Election on 4 July."
```
[the remark this answers is not in the document]
Comment (leenagupte) [reply app/views/components/_global_bar.html.erb:6]: The `if` statement could then be cleaned up in-hours.

[the remark this answers is not in the document]
Comment (AshGDS) [reply app/views/components/_global_bar.html.erb]: Thanks @leenagupte - have added your code (though I altered the date to the 19th) if you're able to approve

[the remark this answers is not in the document]
Comment (leenagupte) [reply app/views/components/_global_bar.html.erb]: Good catch! I'd been playing around with how `before?` and `after?` read and must have gotten my dates mixed up.

**Review by leenagupte — APPROVED** [review]

File: app/views/components/_global_bar.html.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (AshGDS) [reply app/views/components/_global_bar.html.erb]: @leenagupte no worries - thanks!

[the remark this answers is not in the document]
Comment (leenagupte) [reply app/views/components/_global_bar.html.erb]: For anyone worried about this working, the [original PR to add these functions to Rails 6](https://github.com/rails/rails/pull/32185) includes time based tests.

Also, a bit of console testing:

```
irb(main):001> Time.zone.now.before?(Date.new(2024, 6, 19))
=> true
irb(main):002> Date.new(2024, 6, 19).before?(Date.new(2024, 6, 19))
=> false
irb(main):001> t = Time.utc(2024, 6, 19, 0, 0, 0).before?(Date.new(2024, 6, 19))
=> false
```

**Pull request merged**